### PR TITLE
Bugfix/tdi 38351 update axis2 libraries

### PR DESCRIPTION
**Pull request by dmytro-sylaiev**
New libraries have required fixes.

.zip file with new libraries and .pom files for nexus will be sent after merge.

### CODE REVIEW

File: main/plugins/org.talend.designer.components.localprovider/components/tMicrosoftCrmInput/tMicrosoftCrmInput_java.xml [hunk 1]
```diff
@@ -11466,9 +11466,9 @@
...
-                <IMPORT NAME="commons-logging" MODULE="commons-logging-1.1.1.jar" MVN="mvn:org.talend.libraries/commons-logging-1.1.1/6.0.0"  UrlPath="platform:/base/plugins/org.apache.commons.logging_1.1.1.v201101211721.jar" REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')"/>
-                <IMPORT NAME="httpcore-4.2.1" MODULE="httpcore-4.2.1.jar" MVN="mvn:org.talend.libraries/httpcore-4.2.1/6.0.0"  UrlPath="platform:/plugin/org.talend.libraries.apache.http/lib/httpcore-4.2.1.jar" REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')" />
-                <IMPORT NAME="httpclient-4.2.1" MODULE="httpclient-4.2.1.jar" MVN="mvn:org.talend.libraries/httpclient-4.2.1/6.0.0"  UrlPath="platform:/plugin/org.talend.libraries.apache.http/lib/httpclient-4.2.1.jar" REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')" />
+                <IMPORT NAME="commons-logging" MODULE="commons-logging-1.2.jar" MVN="mvn:org.talend.libraries/commons-logging-1.2/6.0.0"  UrlPath="platform:/base/plugins/org.apache.commons.logging_1.2.v201101211721.jar" REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')"/>
```
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tMicrosoftCrmInput/tMicrosoftCrmInput_java.xml]: The UrlPath is wrong, the jar `org.apache.commons.logging_1.2.v201101211721.jar` can't be find in `plugins`
 `org.apache.commons.logging_1.2.0.jar` can be found.

File: main/plugins/org.talend.designer.components.localprovider/components/tMicrosoftCrmOutput/tMicrosoftCrmOutput_java.xml [hunk 2]
```diff
@@ -17119,20 +17118,20 @@
...
             <IMPORT NAME="jaxen-1.1.6" MODULE="jaxen-1.1.6.jar" MVN="mvn:org.talend.libraries/jaxen-1.1.6/6.0.0"  REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')" BundleID="" />
             <IMPORT NAME="mail-1.4" MODULE="mail-1.4.jar" MVN="mvn:org.talend.libraries/mail-1.4/6.0.0"  UrlPath="platform:/plugin/org.talend.libraries.apache.axis2/lib/mail-1.4.jar" REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')" />
-            <IMPORT NAME="neethi-3.0.2" MODULE="neethi-3.0.2.jar" MVN="mvn:org.talend.libraries/neethi-3.0.2/6.0.0"  UrlPath="platform:/plugin/org.talend.libraries.apache.axis2/lib/neethi-3.0.2.jar" REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')" />
+            <IMPORT NAME="neethi-3.0.3" MODULE="neethi-3.0.3.jar" MVN="mvn:org.talend.libraries/neethi-3.0.3/6.0.0"  UrlPath="platform:/plugin/org.talend.libraries.apache.axis2/lib/neethi-3.0.3.jar" REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')" />
```
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tMicrosoftCrmOutput/tMicrosoftCrmOutput_java.xml]: For this kind of jar `neethi-3.0.3.jar`, if we don't upload to talend source code. Then `UrlPath` can be removed.

File: main/plugins/org.talend.designer.components.localprovider/components/tMicrosoftCrmInput/tMicrosoftCrmInput_java.xml [hunk 3]
```diff
@@ -11441,18 +11441,18 @@
...
                 <IMPORT NAME="jaxen-1.1.6" MODULE="jaxen-1.1.6.jar" MVN="mvn:org.talend.libraries/jaxen-1.1.6/6.0.0"  REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')" BundleID="" />
                 <IMPORT NAME="mail-1.4" MODULE="mail-1.4.jar" MVN="mvn:org.talend.libraries/mail-1.4/6.0.0"  UrlPath="platform:/plugin/org.talend.libraries.apache.axis2/lib/mail-1.4.jar" REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')" />
-                <IMPORT NAME="neethi-3.0.2" MODULE="neethi-3.0.2.jar" MVN="mvn:org.talend.libraries/neethi-3.0.2/6.0.0"  UrlPath="platform:/plugin/org.talend.libraries.apache.axis2/lib/neethi-3.0.2.jar" REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')" />
+                <IMPORT NAME="neethi-3.0.3" MODULE="neethi-3.0.3.jar" MVN="mvn:org.talend.libraries/neethi-3.0.3/6.0.0"  UrlPath="platform:/plugin/org.talend.libraries.apache.axis2/lib/neethi-3.0.3.jar" REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')" />
```
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tMicrosoftCrmInput/tMicrosoftCrmInput_java.xml]: For this kind of jar `neethi-3.0.3.jar`, if we don't upload to talend source code. Then `UrlPath` can be removed.

File: main/plugins/org.talend.designer.components.localprovider/components/tMicrosoftCrmInput/tMicrosoftCrmInput_java.xml [hunk 4]
```diff
@@ -11466,9 +11466,9 @@
...
-                <IMPORT NAME="httpcore-4.2.1" MODULE="httpcore-4.2.1.jar" MVN="mvn:org.talend.libraries/httpcore-4.2.1/6.0.0"  UrlPath="platform:/plugin/org.talend.libraries.apache.http/lib/httpcore-4.2.1.jar" REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')" />
-                <IMPORT NAME="httpclient-4.2.1" MODULE="httpclient-4.2.1.jar" MVN="mvn:org.talend.libraries/httpclient-4.2.1/6.0.0"  UrlPath="platform:/plugin/org.talend.libraries.apache.http/lib/httpclient-4.2.1.jar" REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')" />
+                <IMPORT NAME="commons-logging" MODULE="commons-logging-1.2.jar" MVN="mvn:org.talend.libraries/commons-logging-1.2/6.0.0"  UrlPath="platform:/base/plugins/org.apache.commons.logging_1.2.v201101211721.jar" REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')"/>
+                <IMPORT NAME="httpcore-4.4.4" MODULE="httpcore-4.4.4.jar" MVN="mvn:org.talend.libraries/httpcore-4.4.4/6.3.0"  UrlPath="platform:/plugin/org.talend.libraries.apache.http/lib/httpcore-4.4.4.jar" REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')" />
```
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tMicrosoftCrmInput/tMicrosoftCrmInput_java.xml]: Same with ` neethi-3.0.3.jar`. also need update another compoent

File: main/plugins/org.talend.designer.components.localprovider/components/tMicrosoftCrmInput/tMicrosoftCrmInput_java.xml [hunk 5]
```diff
@@ -11466,9 +11466,9 @@
...
+                <IMPORT NAME="httpclient-4.5.2" MODULE="httpclient-4.5.2.jar" MVN="mvn:org.talend.libraries/httpclient-4.5.2/6.3.0"  UrlPath="platform:/plugin/org.talend.libraries.apache.http/lib/httpclient-4.5.2.jar" REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')" />
```
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tMicrosoftCrmInput/tMicrosoftCrmInput_java.xml]: Same with ` neethi-3.0.3.jar`. also need update another compoent

File: main/plugins/org.talend.designer.components.localprovider/components/tMicrosoftCrmInput/tMicrosoftCrmInput_java.xml [hunk 6]
```diff
@@ -11441,18 +11441,18 @@
                 <IMPORT NAME="talend-mscrm-2.0-20170303" MODULE="talend-mscrm-2.0-20170303.jar" MVN="mvn:org.talend.libraries/talend-mscrm-2.0-20170303/6.0.0"  UrlPath="platform:/plugin/org.talend.libraries.crm/lib/talend-mscrm-2.0-20170303.jar" REQUIRED_IF="(AUTH_TYPE=='ONLINE') AND (API_VERSION=='API_2011')" />
                 <!-- axis2 1.7.0 -->
```
Comment (jzhao-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tMicrosoftCrmInput/tMicrosoftCrmInput_java.xml]: change the comment

**Comment by jzhao-talend** [comment]
https://github.com/Talend/tdi-studio-se/blob/88df3bba9b72173bcf5dddacc972d3bdd7d7254d/main/plugins/org.talend.designer.components.libs/libs_src/talend-mscrm
I am not sure whether this client source code need to regenerated with new axis2 lib.
You need confirm with @chmyga

**Comment by dmytro-sylaiev** [comment]
**DO NOT MERGE IT NOW**

PR needs approved changes of another one PR

**Comment by dmytro-sylaiev** [comment]
**Build was successful**, can be merged after review

**Pull request merged**